test/test.nix

```diff
@@ -56,6 +56,9 @@ import ./make-test.nix rec {
 
     services.backups.enable = true;
 
+    services.btcpayserver.enable = true;
+    services.btcpayserver.lightning-node = "lnd";
```

Comment:
Hm, no clightning test?

Reply:
If we implement a clightning test we need to create 4 new tests: btcpayserver lnd with netns, btcpayserver lnd without netns, btcpayserver clightning with netns, btcpayserver clightning without netns. Is this complexity really justified seeing that btcpayserver won't test the connection to `lightning-node` until the node is fully synced anyway?

Reply:
Double-checked that BTCPayServer doesn't connect to Lightning Node until you select the proper node in `General Settings` UI, the reason why I use LND is to test the macaroon creation.

Reply:
Hm, yeah that wouldn't help.